diff --git a/node_modules/react-native-maps-super-cluster/ClusteredMapView.js b/node_modules/react-native-maps-super-cluster/ClusteredMapView.js
index 7d6e7d5..a3b4c9c 100644
--- a/node_modules/react-native-maps-super-cluster/ClusteredMapView.js
+++ b/node_modules/react-native-maps-super-cluster/ClusteredMapView.js
@@ -42,12 +42,12 @@ export default class ClusteredMapView extends PureComponent {
     this.clusterize(this.props.data)
   }
 
-  componentWillReceiveProps(nextProps) {
+  UNSAFE_componentWillReceiveProps(nextProps) {
     if (this.props.data !== nextProps.data)
       this.clusterize(nextProps.data)
   }
 
-  componentWillUpdate(nextProps, nextState) {
+  UNSAFE_componentWillUpdate(nextProps, nextState) {
     if (!this.isAndroid && this.props.animateClusters && this.clustersChanged(nextState))
       LayoutAnimation.configureNext(this.props.layoutAnimationConf)
   }
